common/starbase_buildings/hfe_starbase_building_population.txt

```diff
@@ -22,10 +22,8 @@ hfe_starbase_building_population_navigator = {
 		}
 	}
 	potential = {
-		exists = owner
-		owner = {
-			has_technology = hfe_tech_population_navigator
-		}
+#		exists = owner
```

Comment:
where is reason why I put exists everywhere

Reply:
I apologize. I'll fix that now.

Reply:
Okay, done